java/ql/src/experimental/Security/CWE/CWE-094/InsecureBeanValidation.qhelp

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+
+<overview>
+<p>Bean validation custom constraint error messages support different types of interpolation, including [Java EL expressions](https://docs.jboss.org/hibernate/validator/5.1/reference/en-US/html/chapter-message-interpolation.html#section-interpolation-with-message-expressions).
```

Comment:
Could you perhaps apply a maximum line width of 80-120 or something like that for this entire file?

Reply:
should I include the MD links on that limit considering they shouldnt be visible in VSCode?